packages/mobile/src/screens/chat-screen/ChatListScreen.tsx

```diff
@@ -132,6 +145,10 @@ export const ChatListScreen = () => {
     }
   }, [chats, dispatch])
 
+  const handleLoadMore = useCallback(() => {
```

Comment:
Safe to be called many times? I usually see guards on these to prevent duplicate calls.

Reply:
Good callout.